com.brave.Brave.yaml

```diff
@@ -0,0 +1,117 @@
+app-id: com.brave.Brave
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: brave
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--env=GTK_PATH=/app/lib/gtkmodules'
+  - '--socket=cups'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--own-name=org.mpris.MediaPlayer2.chromium.*'
+  - '--system-talk-name=org.freedesktop.Avahi'
+  - '--system-talk-name=org.freedesktop.GeoClue2'
+  - '--system-talk-name=org.freedesktop.UPower'
+  - '--filesystem=host-etc:ro'
+  - '--filesystem=xdg-download'
```

Comment:
https://github.com/flathub/flathub/pull/1852#discussion_r495574128

Can you try adding

```json
- '--filesystem=/media:ro'
- '--filesystem=/mnt:ro'
```
and check if it works?

Reply:
It would work but not for everybody. Let's be restrictive by default and only allow rw to ~/Downloads and then let the user allow whatever they want with Flatseal

That's what Firefox is doing

Reply:
Why would we want to do something as specific as this?

Reply:
> It would work but not for everybody. Let's be restrictive by default and only allow rw to ~/Downloads and then let the user allow whatever they want with Flatseal
> 
> That's what Firefox is doing

Understandable.